dracut/anaconda-diskroot

```diff
@@ -24,7 +24,7 @@ run_checkisomd5() {
                 rc=$?
                 state="inactive"
             fi
-            if [ "$rc" == "1" ]; then
+            if [ "$rc" = "1" ]; then
```

Comment:
Please, split this commit or at least add explanations to the commit message. I like how it is done in other commits, because it is very easy to follow the changes.

Reply:
Yes, I agree. I was afraid it will be a lot of commit spam, but initially had these changes split.